static/bidder-info/concert.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://bids.concert.io/bids/openrtb"
+endpointCompression: gzip
+geoscope:
+  - global
+maintainer:
+  email: support@concert.io
```

Comment:
@BrettBlox We have sent a verification email to `support@concert.io`. Please reply to that mail with 'Received' to complete the verification process.

Reply:
@Sonali-More-Xandr We just responded to your email.

Reply:
@Sonali-More-Xandr was our response to your email received?

Reply:
email received. thanks!